### PR TITLE
fix jdk9 build

### PR DESCRIPTION
**Pull request by pdex**

### CODE REVIEW

File: xio-core/pom.xml [hunk 1]
```diff
@@ -243,22 +250,29 @@
...
+      <artifactId>powermock-core</artifactId>
+      <version>2.0.0-beta.5</version>
       <scope>test</scope>
     </dependency>
 
    <dependency>
       <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito</artifactId>
-      <version>1.6.5</version>
+      <artifactId>powermock-api-mockito2</artifactId>
```
Comment (manimaul) [inline xio-core/pom.xml]: i wonder if we still need powermock - https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#unmockable

File: README.md [hunk 2]
```diff
@@ -49,3 +49,17 @@ following maven plugins maintain the source code standards:
...
+
+### Testing new jdk versions
+
+#### build the docker image
+
+docker build -t xio-jdk9 .
+
+#### test the build
+
+docker run  --mount type=bind,src=`pwd`,dst=/home -t xio-jdk9 ./mvnw verify
```
Comment (manimaul) [inline README.md]: i think the recommended way to do this now is with --volume rather than --mount

Comment (pdex) [reply README.md]: /shrug docker

File: xio-core/pom.xml (hunk 1 above)
Comment (pdex) [reply xio-core/pom.xml]: I'm fine with that, we should probably have some guidelines around when/how we mock things.

**Pull request merged**